libs/eo/consent/data-access-api/consent.service.ts

```diff
@@ -31,6 +31,12 @@ export interface EoConsent {
   idpClientId: string;
 }
 
+export interface EoReceivedConsent {
```

Comment:
Eo is so yesterday.

Reply:
True dat, however the rename has not been made so they are still named eo to avoid usage of both eo and ett